### PR TITLE
trivial: Fixed typos and cleaned up language

### PR DESCRIPTION
**Pull request by fanquake**
This rebases and fixes some of the outstanding nits in #13010. Let either merge quickly or close for now.

### CODE REVIEW

**Comment by practicalswift** [comment]
utACK modulo fixing:

```
This diff appears to have added new lines with trailing whitespace.
The following changes were suspected:
diff --git a/doc/tor.md b/doc/tor.md
@@ -104,9 +105,9 @@ To show verbose debugging information, pass `-debug=tor`.
+are seen with `-debug=tor` they can be resolved by adding both the user running Tor and
^---- failure generated from contrib/devtools/lint-whitespace.sh
```

**Comment by Empact** [comment]
I think there's some language confusion in the changes. E.g. Bitcoin is a proper noun for the currency / project / network, hence capitalized. "bitcoin" is used as shorthand for Bitcoin Core and arguably serves well lower-case. Bitcoin-Qt is the actual executable name of the wallet software, and is capitalized like so in its naming.

File: doc/README.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ Bitcoin Core
 
 Setup
 ---------------------
-Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions (which is currently more than 100 GBs); depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
+Bitcoin Core, the original bitcoin client, builds the network's backbone. It downloads and, by default, stores the entire history of bitcoin transactions (which is currently more than 160 GB). Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to over a day.
```
Comment (Empact) [inline doc/README.md]: I like the author's suggestion of a different first sentence, which could be integrated: "Bitcoin Core is the original and current reference implementation of the Bitcoin network" https://github.com/bitcoin/bitcoin/pull/13010#discussion_r184677201

**Comment by MarcoFalke** [comment]
I guess the `bitcoin` -> `Bitcoin Core` replacements make sense. The other changes are hard to review for non-native speakers, so I'd suggest dropping them.

**Comment by sipa** [comment]
I think we generally use 'bitcoin' to refer to the currency, and "Bitcoin" for the project/network/technology.

**Comment by fanquake** [comment]
Rebased, dropped the more verbose changes from the first commit, and cherry picked another related commit.

**Comment by practicalswift** [comment]
ACK 756d4d38b21f09f76b3d6fcf7d9189345ea834ba

**Comment by Varunram** [comment]
Thanks, looks good.

File: doc/tor.md [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
-It is possible to run Bitcoin as a Tor hidden service, and connect to such services.
+It is possible to run Bitcoin Core as a Tor hidden service, and connect to such services.
 
 The following directions assume you have a Tor proxy running on port 9050. Many distributions default to having a SOCKS proxy listening on port 9050, but others may not. In particular, the Tor Browser Bundle defaults to listening on port 9150. See [Tor Project FAQ:TBBSocksPort](https://www.torproject.org/docs/faq.html.en#TBBSocksPort) for how to properly
 configure Tor.
 
 
-## 1. Run bitcoin behind a Tor proxy
+## 1. Run Bitcoin Core behind a Tor proxy
+---------------------------------
```
Comment (MarcoFalke) [inline doc/tor.md]: Unrelated `------------------------`

**Review by maflcko — COMMENTED** [review]
utACK 756d4d38b21f09f76b3d6fcf7d9189345ea834ba

File: doc/tor.md (hunk 2 above)
Comment (fanquake) [reply doc/tor.md]: Thanks, fixed.

**Comment by practicalswift** [comment]
ACK d5f745a5c76ad5fbcc9a20fe128806e39de52a2c

**Comment by promag** [comment]
ACK d5f745a without squash to keep authors.
